app/build.gradle

```diff
@@ -3,6 +3,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'org.jlleitschuh.gradle.ktlint'
+apply plugin: 'com.google.gms.google-services'
```

Comment:
@bstrauch24 Turns out the firebase service wasn't even starting because this line was missing. I put the logic back how it was, but moved it into it's own manager class to not have so much happening in the MainActivity.